Library/Homebrew/formula_installer.rb

```diff
@@ -407,7 +407,8 @@ def install_dependencies(deps)
     if deps.empty? && only_deps?
       puts "All dependencies for #{formula.full_name} are satisfied."
     else
-      oh1 "Installing dependencies for #{formula.full_name}: #{Tty.green}#{deps.map(&:first)*", "}#{Tty.reset}" unless deps.empty?
+      oh1 "Installing dependencies for #{formula.full_name}"
```

Comment:
@UniqMartin Would you like to submit a PR with the above code, or shall I add it to this PR?